examples/influxdb-ami/README.md

```diff
@@ -75,6 +74,4 @@ local checkout of the Module, uploaded from your own computer via the `file` pro
 
 The Packer template in this example folder can build not only AMIs, but also Docker images for local testing. This is
 convenient for testing out the various scripts in the `modules` folder without having to wait for an AMI to build and
-a bunch of EC2 Instances to boot up. See the [influxdb-docker 
```

Comment:
Should link to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/local-mocks